example/20-componentconfig-gardener-controller-manager.yaml

```diff
@@ -31,6 +31,16 @@ controllers:
     staleGracePeriodDays: 14
     staleExpirationTimeDays: 90
     staleSyncPeriod: 12h
+  # quota:
+  #   projectSelector: {}
```

Comment:
> If we allow the administrator to specify a projectSelector for the default quota, would it make sense to allow specify a set/array of default quotas for different label selectors?

Okay, will do.

> Otherwise, if the projectSelector is only there to allow excluding some projects from the default quota: should we rather remove the projectSelector and use a well-defined label for excluding projects from the default quota?

You can use it for exclusion, but you are not bound to it. That's the nice and flexible part about label selectors and the main reason I introduced them here.